www/src/components/email-capture-form.js

```diff
@@ -1,28 +1,220 @@
 import React from "react"
-import { rhythm } from "../utils/typography"
-import { colors } from "../utils/presets"
-import HubspotForm from "./hubspot-form"
+import { rhythm, options } from "../utils/typography"
+import presets, { colors } from "../utils/presets"
+import hex2rgba from "hex2rgba"
+import { formInput } from "../utils/form-styles"
+import { buttonStyles } from "../utils/styles"
+
+const Label = props => (
+  <label htmlFor={props.for}>
```

Comment:
Does React complain about this in dev mode? Didn't know if `for` may be a reserved prop on _any_ component!